docs/source/notes/get_start_xpu.rst

```diff
@@ -8,68 +8,52 @@ Hardware Prerequisite
    :widths: 50 50
    :header-rows: 1
 
-   * - Validated Hardware
-     - Supported OS
-   * - Intel® Data Center GPU Max Series
-     - Linux
-   * - Intel Client GPU
-     - Windows/Linux
+   * - Supported OS
+     - Validated Hardware
+   * - Linux
+     - Intel® Client GPUs / Intel® Data Center GPU Max Series
+   * - Windows
+     - Intel® Client GPUs
+   * - WSL2 (experimental feature)
+     - Intel® Client GPUs
 
-Intel GPUs support (Prototype) is ready in PyTorch* 2.5 for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios.
+Intel GPUs support (Prototype) is ready in PyTorch* 2.6 for Intel® Client GPUs and Intel® Data Center GPU Max Series on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios.
 
 Software Prerequisite
 ---------------------
 
-Visit `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ for more detailed information regarding:
+To use PyTorch on Intel GPUs, you need to install the Intel GPUs driver first. For installation guide, visit `Intel GPUs Driver Installation <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu/2-6.html#driver-installation>`_.
+
+Installation from binaries only need Intel GPU Drivers, while building from source requires Intel GPU Drivers and Intel® Deep Learning Essentials. Please refer to  `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu/2-6.html>`_.
```

Comment:
Also, I'm not sure what are the policy about those ® signs, but I had a feeling it should be used for the first mention, rather than the last one.
```suggestion
Intel GPU Drivers are sufficient for binary installation, while building from source requires both Intel GPU Drivers and Intel® Deep Learning Essentials. Please refer to  `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu/2-6.html>`_ for more information.
```